src/shape/orientated-box.js

```diff
@@ -0,0 +1,93 @@
+pc.extend(pc, function () {
+    var transformedRay = new pc.Ray();
```

Comment:
Please consider using short `tmpRay`, `tmpVec` etc. naming convention here. As this might not get simplified when engine is minified, leading to unnecessary extra size.